app/collections/show.cjsx

```diff
@@ -8,11 +8,15 @@ Avatar = require '../partials/avatar'
 Loading = require '../components/loading-indicator'
 HandlePropChanges = require '../lib/handle-prop-changes'
 TitleMixin = require '../lib/title-mixin'
+classNames = require 'classNames'
 
 counterpart.registerTranslations 'en',
   collectionPage:
     settings: 'Settings'
     collaborators: 'Collaborators'
+    collectionsLink: '%(user)s\'s\u00a0Collections'
+    favoritesLink: '%(user)s\'s\u00a0Favorites'
+    userLink: '%(user)s\'s\u00a0Profile'
```

Comment:
Got it. Still don't think `\u00a0` is necessary. This could be achieved by `whitespace: nowrap` in the css.